### PR TITLE
Proposal for issue #438

### PR DESCRIPTION
**Pull request by johnnynanjiang**

### CODE REVIEW

**Comment by vikmeup** [comment]
Message me on telegram: vikmeup

**Comment by vikmeup** [comment]
Can you fix swiftlint warning? I will merge it once I wake up

**Comment by johnnynanjiang** [comment]
Fixed most of the SwiftLint warnings, except the following on, any better idea?

SettingsViewController.swift:53:14: warning: Function Body Length Violation: Function body should span 80 lines or less excluding comments and whitespace: currently spans 153 lines (function_body_length)

File: Trust/Settings/Types/Config.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ struct Config {
         static let isDebugEnabled = "isDebugEnabled"
         static let currencyID = "currencyID"
         static let dAppBrowser = "dAppBrowser"
+        static let testNetworkWarningDontShowAgain = "testNetworkWarningDontShowAgain"
```
Comment (hewigovens) [inline Trust/Settings/Types/Config.swift]: Any other shorter variable name?

File: Trust/Settings/ViewControllers/SettingsViewController.swift [hunk 2]
```diff
@@ -70,8 +70,36 @@ class SettingsViewController: FormViewController, Coordinator {
...
             }.onChange {[weak self] row in
-                self?.config.chainID = row.value?.chainID ?? RPCServer.main.chainID
-                self?.run(action: .RPCServer)
+                let runSelectedNetwork = { [weak self, unowned row] in
+                    self?.config.chainID = row.value?.chainID ?? RPCServer.main.chainID
+                    self?.run(action: .RPCServer)
+                }
+
+                let popupWarningMessage = { [weak self] in
+                    let alertViewController = UIAlertController(title: NSLocalizedString("settings.network.test.warnning.title", value: "Warning", comment: ""),
```
Comment (hewigovens) [inline Trust/Settings/ViewControllers/SettingsViewController.swift]: We can move these localized strings to view model?

File: Trust/Settings/ViewControllers/SettingsViewController.swift [hunk 3]
```diff
@@ -213,7 +241,7 @@ class SettingsViewController: FormViewController, Coordinator {
                 row.cellStyle = .value1
                 row.presentationMode = .show(controllerProvider: ControllerProvider<UIViewController>.callback {
                     return SupportViewController()
-            }, onDismiss: { _ in })
+                }, onDismiss: { _ in })
```
Comment (hewigovens) [inline Trust/Settings/ViewControllers/SettingsViewController.swift]: extra white spaces

**Comment by hewigovens** [comment]
All the git commit messages are same (Bug/438)... Could you please squash them into a single meaningful commit?

**Pull request merged**